scalafmt-cli/jvm/src/main/scala/org/scalafmt/cli/CliOptionsUtils.scala

```diff
@@ -0,0 +1,13 @@
+package org.scalafmt.cli
+
+class CliOptionsUtils {
```

Comment:
why not define a method getConsoleWriter, which returns nullable Writer, or Option[Writer], and the rest of the code is exactly the same and doesn't need to be platform-specific and fully duplicated?